bin/convert-argv.js

```diff
@@ -28,11 +28,23 @@ module.exports = function(optimist, argv, convertOptions) {
 	}
 
 	var configPath, ext;
+	var extensions = Object.keys(interpret.extensions).sort(function(a, b){
+		return a.length - b.length;
+	});
+
 	if (argv.config) {
 		configPath = path.resolve(argv.config);
-		ext = path.extname(configPath);
+		for (var i = extensions.length - 1; i >= 0; i--) {
+			var tmpExt = extensions[i];
+			if (configPath.indexOf(tmpExt, configPath.length - tmpExt.length) > -1){
+				ext = tmpExt;
+				break;
+			}
+		};
+		if (!ext) {
+			ext = path.extname(configPath);
```

Comment:
Why doing `ext = path.extname(configPath);`?

Doesn't this replaces `ext` with `".js"` when `ext === ".babel.js"`?